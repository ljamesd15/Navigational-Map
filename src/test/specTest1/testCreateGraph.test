# testCreateGraph.test file
# Tests that a graph can be created.

# Creates 3 different graphs.
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3