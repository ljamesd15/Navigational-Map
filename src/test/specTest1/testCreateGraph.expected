# testCreateGraph.test file
# Tests that a graph can be created.

# Creates 3 different graphs.
created graph graph1
created graph graph2
created graph graph3